js/dropdown.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
+    $('[data-toggle="dropdown-with-sub-menu"]').click(function(event) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/eyinwag/calico/pullRequest?prid=8501747)
```suggestion
    $("[data-toggle=\"dropdown-with-sub-menu\"]").click(function(event) {
```